### ISSUE TITLE
XML documentation for enum member shall not have a <remarks> section 

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#305]
### Problem statement
Today, the tools such as Sandcastle  handle `remarks` sections by creating own sections in the HTML/CHM.

This works for types, methods, properties, etc. but unfortunately, for enum members it does not work. The resulting output does not contain any Remarks section.

However, the information is available to tools like ReSharper enhanced tooltip or IntelliSense. Thus the information itself is there.

So, as developers may use the HTML/CHM version only when working with 3rd party frameworks, the information of the `<remarks>` section is typically not available to the developers.

### Proposed solution
Therefore, to avoid frustration, we should report an issue if an enum member contains a `<remarks>` section and state that the information shall be available also within the `<summary>` section.

### Example
```C#
public enum MyEnum
{
    /// <summary>
    /// My summary.
    /// </summary>
    /// <remarks>
    /// My remarks.
    /// </remarks>
   MyValue = 0,
}
```

In the example above, `My remarks.` is not available to the developers.
